README.adoc

```diff
@@ -555,7 +561,7 @@ Where:
 
 Note: A helper script is provided for the generation of packages for a specific git commit. For instance:
 
-    $ sh dockerfiles/packaging/guice/cassandra/package.sh 3.2.0 1 c298195e84 $PWD/result
+    $ sh dockerfiles/packaging/guice/cassandra/package.sh 3.5.0 1 c298195e84 $PWD/result
```

Comment:
unneeded change, I guess there is a few chance that c298195e84 is matching 3.5.0 version.